torch/linalg/__init__.py

```diff
@@ -140,6 +140,111 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+svd = _add_docstr(_linalg.linalg_svd, r"""
+linalg.svd(input, full_matrices=True, compute_uv=True, out=None) -> (Tensor, Tensor, Tensor)
+
+This function returns a namedtuple ``(U, S, Vh)`` which is the singular value
+decomposition of a input real matrix or batches of real matrices :attr:`input` such that
+:math:`input = U \times diag(S) \times V^H` (where :math:`V^H` is ``Vh``).
+
+.. warning:: **Differences with** :meth:`~torch.svd`:
```

Comment:
I would remove most of this warning and replace it with a note that just references the NumPy function:

.. Note:: Unlike NumPy's linalg.svd, this always returns a namedtuple of three tensors, even when :attr:\`compute_uv=False\`.

Reply:
> I would remove most of this warning 

why? As I user, if I find two functions with slightly different names, the first thing I wonder is what are the differences between the two, so I think we should document them.

Also: do we plan to deprecate `torch.svd` in favor of `torch.linalg.svd` at some point? If so, we should probably add a note in `torch.svd` to use the linalg version instead.